### PR TITLE
feat: Improve the `README.md` file

### PR DESCRIPTION
**Pull request by Panquesito7**
### Describe the changes you've made

- Improve the `README.md` file.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

I don't think this applies here.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

I don't think this applies here.

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

File: README.md
Comment (shuklaritvik06) [inline README.md:30]: Small typo, should be Now :) , LGTM but we should wait for @vinzvinci review before merging :)

Comment (Panquesito7) [reply README.md:30]: Great catch, thank you! It's fixed.

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :) Let's wait for vinz review :)

**Review by vinzvinci — APPROVED** [review]
LGTM 👍 Also thank you for making it short lol

**Pull request merged**
